src/pages/RepoPage/BundlesTab/BundleContent/AssetsTable/AssetsTable.spec.tsx

```diff
@@ -292,8 +290,14 @@ describe('AssetsTable', () => {
       setup({ isEmptyBundles: true })
       render(<AssetsTable />, { wrapper })
 
-      const table = await screen.findByText('EmptyTable')
-      expect(table).toBeInTheDocument()
+      const tableRoles = await screen.findAllByRole('row')
+      expect(tableRoles).toHaveLength(2)
+
+      const tableCells = await within(tableRoles[1]!).findAllByRole('cell')
```

Comment:
Okay I needed `within` in my life and didn't even know it thanks

Reply:
it is **super** useful